source/module_io/write_wfc_r.cpp

```diff
@@ -31,157 +30,173 @@ void write_psi_r_1(const psi::Psi<std::complex<double>>& wfc_g,
     const std::string outdir = PARAM.globalv.global_out_dir + folder_name + "/";
     ModuleBase::GlobalFunc::MAKE_DIR(outdir);
 #ifdef __MPI
-		std::vector<MPI_Request> mpi_requests;
+    std::vector<MPI_Request> mpi_requests;
 #endif
-		for(int ik=0; ik<wfc_g.get_nk(); ++ik)
-		{
-			wfc_g.fix_k(ik);
-			const int ik_out = (PARAM.inp.nspin!=2)
-				? ik + GlobalC::Pkpoints.startk_pool[GlobalV::MY_POOL]
-				: ik - kv.get_nks()/2*kv.isk[ik] + kv.get_nkstot()/2*kv.isk[ik] + GlobalC::Pkpoints.startk_pool[GlobalV::MY_POOL];
-			for(int ib=0; ib<wfc_g.get_nbands(); ++ib)
-			{
-				const std::vector<std::complex<double>> wfc_r = cal_wfc_r(wfcpw, wfc_g, ik, ib);
-
-                std::vector<double> wfc_r2(wfc_r.size());
-                std::vector<double> wfc_i2;
-                if (square)
-                    for (int ir = 0; ir < wfc_r2.size(); ++ir)
-                        wfc_r2[ir] = std::norm(wfc_r[ir]);   // "std::norm(z)" returns |z|^2
-                else
+    for (int ik = 0; ik < wfc_g.get_nk(); ++ik)
+    {
+        wfc_g.fix_k(ik);
+        const int ik_out = (PARAM.inp.nspin != 2)
+                               ? ik + GlobalC::Pkpoints.startk_pool[GlobalV::MY_POOL]
+                               : ik - kv.get_nks() / 2 * kv.isk[ik] + kv.get_nkstot() / 2 * kv.isk[ik]
+                                     + GlobalC::Pkpoints.startk_pool[GlobalV::MY_POOL];
+        for (int ib = 0; ib < wfc_g.get_nbands(); ++ib)
+        {
+            const std::vector<std::complex<double>> wfc_r = cal_wfc_r(wfcpw, wfc_g, ik, ib);
+
+            std::vector<double> wfc_r2(wfc_r.size());
+            std::vector<double> wfc_i2;
+            if (square)
+            {
+                for (int ir = 0; ir < wfc_r2.size(); ++ir)
                 {
-                    wfc_i2.resize(wfc_r.size());
-                    for (int ir = 0; ir < wfc_r2.size(); ++ir)
-                    {
-                        wfc_r2[ir] = wfc_r[ir].real();
-                        wfc_i2[ir] = wfc_r[ir].imag();
-                    }
+                    wfc_r2[ir] = std::norm(wfc_r[ir]); // "std::norm(z)" returns |z|^2
                 }
-				const std::string file_name = outdir + "wfc_realspace_"
-					+ ModuleBase::GlobalFunc::TO_STRING(ik_out)
-					+ "_" + ModuleBase::GlobalFunc::TO_STRING(ib);
+            }
+            else
+            {
+                wfc_i2.resize(wfc_r.size());
+                for (int ir = 0; ir < wfc_r2.size(); ++ir)
+                {
+                    wfc_r2[ir] = wfc_r[ir].real();
```

Comment:
please do not use letter "r" to represent "real space" and "real part" at the same time...this cause you use "r2" which will further be misleading as "squared" or "multiplied with r^2"...
"wfc_real" is okay